src/awkward/_v2/_connect/numba/builder.py

```diff
@@ -396,6 +410,23 @@ def lower_real(context, builder, sig, args):
     return context.get_dummy_value()
 
 
+@numba.extending.lower_builtin("complex", ArrayBuilderType, numba.types.Complex)
+def lower_complex(context, builder, sig, args):
+    arraybuildertype, xtype = sig.args
+    arraybuilderval, xval = args
+    proxyin = context.make_helper(builder, arraybuildertype, arraybuilderval)
+    z_imag = xval.operands.pop()
+    z_real = xval.operands.pop().value
```

Comment:
```suggestion
    z = context.make_complex(builder, xtype, xval)
    z_real, z_imag = z.real, z.imag
```

I don't know if you need `z.real._getvalue()` or not...

Reply:
done